koku/masu/database/reporting_common_db_accessor.py

```diff
@@ -20,7 +20,7 @@
 import django.apps
 
 from masu.database.koku_database_access import KokuDBAccess
-from reporting_common.models import ReportColumnMap
+from reporting_common import REPORT_COLUMN_MAP
 
 
 class ReportingCommonDBAccessor(KokuDBAccess):
```

Comment:
I don't think this is the right place for this to live since it is not in a database.

Reply:
@kholdaway where would you like to put it?

Reply:
I will let you figure that out.  But a DB accessor is likely not a good fit.

Reply:
@kholdaway Yes, I'll do as you asked:

To change this I'll edit 45 places in the code that do this:
```
        with ReportingCommonDBAccessor() as reporting_common:
            self._column_map = reporting_common.column_map
```

to do this instead:

```
self._column_map = REPORTING_COMMON_MAP
```

Reply:
It sounds like trivial changes to access a constant.  In general, the DB accessor would be expected to access the DB.  This PR is moving a function from a DB to a python constant.  Relocating it would provide clarity in the code.

Reply:
These lines don't access the DB.  https://github.com/project-koku/koku/blob/28723001fa29df277a6d4e4f4262845921fd70f4/koku/masu/database/reporting_common_db_accessor.py#L66-L76

Reply:
To summarize what we discussed on the bluejeans call.
1. What benefit does generate_column_map give us now?  There is no DB to transform.  Why not just store it in the right format to start with?
2. Why does generate_column_map need to run every time a ReportingCommonDBAccessor is created if the output is always the same?
3. Why do other classes need to create a ReportingCommonDBAccessor class only to access a constant located in the  reporting_common/__init__.py module?  That's a lot of waste creating objects and disposing of them plus garbage collection.  Access the constant directly.
4. Can all this function just live in  reporting_common/__init__.py and accessors us that directly?

Reply:
@kholdaway I've added it. Please re-review.

Reply:
Thanks